packages/core/src/mpi/merge-patients.ts

```diff
@@ -0,0 +1,11 @@
+import { Patient } from "../domain/patient";
+import { PatientMPI } from "./shared";
+
+export type MergeProtocol = (patients: Patient[], currentPatient?: Patient) => Patient | undefined;
+
+export function useFirstMatchingPatient(patients: PatientMPI[]): PatientMPI | undefined {
+  if (patients.length === 0) return undefined;
+  if (patients.length === 1) return patients[0];
+
+  return patients[0];
```

Comment:
NABD, but just `return patients[0];` would do - JS/TS doesn't fail if the position doesn't exist in the array, it just returns `undefined`.